CHANGELOG.md

```diff
@@ -11,6 +11,13 @@
 - Fix UI issues in multi nodes environment when a node
   is unavailable (PR[#3521](https://github.com/scality/metalk8s/pull/3521))
 
+## Bug fixes
+
+- Manage Control Plane Ingress Virtual IP with MetalLB is not
+  supported when you have multiple subnet in Control Plane
+  network, now it's properly checked
```

Comment:
Not 100% convinced about the sentence :smile: :thinking:

Reply:
We usually document the change rather than the issue in this document, so maybe:

```suggestion
- Enforce a single subnet for control plane when using a
  MetalLB-managed VIP for Ingress
```

And maybe add a link to the relevant issue and/or discussions you had with the MetalLB guys when investigating this limitation.

Reply:
Ok fair,
For the link I'm not sure it's needed, we have the link to the PR that have a link to the ticket that talk about it, not sure we want to put the link to the ticket directly in the changelog